docs/source/postprocessing.rst

```diff
@@ -0,0 +1,471 @@
+   :header-rows: 1
+
+   * - Category
+     - Variable
+     - Description
+   * - Displacements
+     - u
+     - Scalar displacement
+   * - Strains
+     - ux, uy (2D) / ux, uy, uz (3D)
+     - Gradient components
+   * - Stresses
+     - sx, sy (2D) / sx, sy, sz (3D)
+     - Flux components
+   * - Extras
+     - failure
+     - Failure index
+   * - 
+     - dv1, dv2, dv3
+     - Design variables
+   * - Loads
+     - f
+     - Applied load
+
+PCM Elements (TACS_PCM_ELEMENT)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table:: PCM Element Output Variables
+   :widths: 20 20 60
+   :header-rows: 1
+
+   * - Category
+     - Variable
+     - Description
+   * - Displacements
+     - dT
+     - Temperature change
+   * - Strains
+     - gradx, grady
+     - Temperature gradient components
+   * - Stresses
+     - fluxx, fluxy
+     - Heat flux components
+   * - Extras
+     - rho
+     - Density
+   * - 
+     - dv1, dv2, dv3
+     - Design variables
+   * - 
+     - phase
+     - Phase field
+   * - Loads
+     - Q
+     - Applied heat source
+
+Visualization Tips
+------------------
+
+1. **Element-wise vs. Nodal Data**: F5 files contain both element-wise and nodal data. The conversion utilities automatically perform nodal averaging for element-wise quantities.
+
+2. **Higher-order Elements**: Higher-order elements are automatically converted to basic element types for visualization (e.g., quadratic triangles become linear triangles).
+
+3. **Component Separation**: In Tecplot, each component in the model can be written as a separate zone in the output files, making it easy to visualize different parts of the structure.
+
+4. **Time-dependent Data**: Use the ``--use_strands`` option with ``f5totec`` for time-dependent analyses to create animated visualizations.
+
+5. **Large Models**: For very large models, consider using only the essential output flags to reduce file size and processing time.
+
+Visualizing Deformed Surfaces
+-----------------------------
+
+One of the most common postprocessing tasks is visualizing the deformed shape of structures. TACS provides both nodal coordinates (X, Y, Z) and displacements (u, v, w) that can be used to create deformed surface visualizations.
+
+Creating Deformed Geometry in Tecplot
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In Tecplot, you can visualize deformed surfaces by creating new variables that represent the deformed coordinates:
+
+1. **Open the converted .plt file** in Tecplot
+2. **Create new variables** for deformed coordinates:
+   - Go to ``Data > Alter > Specify Equations``
+   - Create new variables:
+     - ``{XDEF} = {X} + {u}`` (deformed X coordinate)
+     - ``{YDEF} = {Y} + {v}`` (deformed Y coordinate)  
+     - ``{ZDEF} = {Z} + {w}`` (deformed Z coordinate)
+3. **Create the deformed plot**:
+   - Go to ``Plot > 3D``
+   - Set ``X``, ``Y``, ``Z`` to ``XDEF``, ``YDEF``, ``ZDEF``
+   - Choose appropriate surface rendering (``Surface``, ``Mesh``, or ``Contour``)
```

Comment:
This is irrelevant to plotting the deformed surface

Reply:
I assume you mean the 3rd bullet? Could you clarify @A-CGray

Reply:
Nevermind, gotcha. I'll update this to match Tecplots actual UI